config/puma.rb

```diff
@@ -0,0 +1,9 @@
+workers Integer(ENV['WEB_CONCURRENCY'] || 2)
+threads_count = Integer(ENV['MAX_THREADS'] || 5)
```

Comment:
Well, I'm just gonna merge away and see how this goes. :grimacing: In case something breaks we'll give it a whack with then ENV variables. :hammer_and_pick: